packages/pyroscope-flamegraph/src/convert/sandwichViewProfiles.ts

```diff
@@ -170,12 +168,15 @@ export function callersFlamebearer(
   processTree(tree);
 
   subtrees.forEach((v, i) => {
-    totalNode.children.push(
-      arrayToTree(v, maxSubtreeLvl, targetFunctionTotals[i])
-    );
+    totalNode.children.push(arrayToTree(v.reverse(), targetFunctionTotals[i]));
```

Comment:
Major change I made here is that I first make an inverse (upside down) tree.
This allows me to call `dedupTree` function on the whole structure.
And then I reverse it back on line 177.

Reply:
This allows us to get rid of some other code, e.g all that `maxSubtreeLvl` logic

Reply:
i don't think that's obvious, a comment in the code would be appreciated